.github/workflows/validate.yml

```diff
@@ -2,8 +2,8 @@ name: "Validate"
 
 on:
   workflow_dispatch:
-  schedule:
-    - cron:  "0 0 * * *"
```

Comment:
This schedule could be added back in, but it was disabled in my fork for having this.